build/submodules.props

```diff
@@ -37,6 +37,6 @@
 
   <ItemGroup>
     <Repository Include="Templating" PatchPolicy="AlwaysUpdateAndCascadeVersions" RootPath="$(RepositoryRoot)src\Templating\" />
```

Comment:
I think we always keep Templating as `Repository` and not move it into `ShippedRepository` is that correct? Also, I'm assuming we don't have to do any updates in the Templating repo itself?